### PR TITLE
Release 0.4.0

### PR DESCRIPTION
**Pull request by SuperQ**
* [CHANGE] Replace legacy common/log with promlog #85
* [FEATURE] Add current recursive clients metric #74
* [FEATURE] Add zone serial numbers as metrics #91
* [BUGFIX] Use uint64 for counters in v3 xml. #70
* [BUGFIX] Fix Gauge type for large gauges #90

Signed-off-by: Ben Kochie <superq@gmail.com>

### CODE REVIEW

**Comment by SuperQ** [comment]
Waiting for https support to be added.

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,6 +1,13 @@
 ## master / unreleased
+
+## 0.4.0 / 2021-01-12
   
+* [CHANGE] Replace legacy common/log with promlog #85
+* [FEATURE] Add current recursive clients metric #74
+* [FEATURE] Add zone serial numbers as metrics #91
+* [FEATURE] Add TLS and basic authentication #94
 * [BUGFIX] Use uint64 for counters in v3 xml. #70
```
Comment (roidelapluie) [inline CHANGELOG.md:9]:
```suggestion
* [BUGFIX] Use uint64 for counters in v3 xml #70
```

Comment (SuperQ) [reply CHANGELOG.md]: Fixed

**Review by roidelapluie — APPROVED** [review]

**Comment by roidelapluie** [comment]
If you want you can fix the date in the changelog.

**Pull request merged**